projects/adf4368/src.mk

```diff
@@ -0,0 +1,47 @@
+include $(PROJECT)/src/platform/$(PLATFORM)/platform_src.mk
+include $(PROJECT)/src/examples/examples_src.mk
+
+NO_OS_INCS_DIR += $(INCLUDE)
+
+SRCS += $(PROJECT)/src/platform/$(PLATFORM)/main.c
+
+INCS += $(PROJECT)/src/common/common_data.h
+SRCS += $(PROJECT)/src/common/common_data.c
+
+INCS += $(PROJECT)/src/platform/platform_includes.h
+
+INCS += $(PROJECT)/src/platform/$(PLATFORM)/parameters.h
+SRCS += $(PROJECT)/src/platform/$(PLATFORM)/parameters.c
+
+SRCS += $(DRIVERS)/api/no_os_uart.c     \
+        $(DRIVERS)/api/no_os_gpio.c     \
+	$(DRIVERS)/api/no_os_spi.c  	\
+	$(DRIVERS)/api/no_os_irq.c 	\
+	$(DRIVERS)/api/no_os_timer.c	\
+        $(DRIVERS)/api/no_os_dma.c      \
+        $(NO-OS)/util/no_os_fifo.c      \
+	$(NO-OS)/util/no_os_mutex.c     \
+	$(NO-OS)/util/no_os_lf256fifo.c \
+        $(NO-OS)/util/no_os_list.c      \
+        $(NO-OS)/util/no_os_util.c      \
+        $(NO-OS)/util/no_os_alloc.c
+
+INCS += $(NO_OS_INCS_DIR)/no_os_delay.h     \
```

Comment:
you did this above:
```
NO_OS_INCS_DIR += $(INCLUDE)
```

so delete all thise section, line 29 to 44

Reply:
when I delete them it does not compile